apps/blockchain/scripts/generate_state_tests.ex

```diff
@@ -8,49 +8,49 @@ defmodule GenerateStateTests do
   use EthCommonTest.Harness
 
   @hardforks ["EIP158", "EIP150", "Homestead", "Frontier", "Byzantium"]
-
-  def run(_args) do
-    initial_state = %{
-      passing: %{
-        "Homestead" => [],
-        "Frontier" => [],
-        "EIP150" => [],
-        "EIP158" => [],
-        "Byzantium" => []
-      },
-      failing: %{
-        "Homestead" => [],
-        "Frontier" => [],
-        "EIP150" => [],
-        "EIP158" => [],
-        "Byzantium" => []
-      }
+  @twenty_minutes 60 * 20 * 1000
```

Comment:
What do you think about pulling these changes into its own PR? Since they seem like two independent pieces of work, I think it would make sense to include them separately.

Reply:
@germsvel https://github.com/poanetwork/mana/pull/381